pkg/synopsysctl/cmd_root.go

```diff
@@ -61,8 +61,7 @@ var rootCmd = &cobra.Command{
 		}
 		// Sets kubeconfig and initializes resource client libraries
 		if err := setResourceClients(); err != nil {
-			log.Error(err)
-			os.Exit(1)
+			log.Warnf("resource clients were not set properly, %v", err)
```

Comment:
I don't think we want to log anything in the case of the native command.  Printing even a warning would seem confusing imo.  May need to take most of PreRunE and place in a function that only non-native commands would use unless there is a better way to have the native sub-commands not use this PreRunE (which I'm not sure there is)

Reply:
There is a hack where I can have a `PersistentRunE` for a subcommand that will override the root command, but it's being fixed in cobra right now.  See issue [252](https://github.com/spf13/cobra/issues/252)